ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/summer/.content.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : why doe we need this segment ?

Reply:
deleted.